Docker deploy error

I am planning to use SQLite as the database, here is my app.conf

```
driverName = sqlite
dataSourceName = "file:/db/casibase.db?cache=shared"
dbName = casibase
```

`docker-compose.yaml` content is:

```
version: '3'

services:
 casbin:
   container_name: casibase
   image: casbin/casibase:latest
   restart: always
   volumes:
     - ./config/app.conf:/conf/app.conf
     - ./db/casibase.db:/db/casibase.db
   ports:
     - "14000:14000"
```

Error occurred after startup:

```
goroutine 1 [running]:
github.com/casibase/casibase/object.(*Adapter).createTable(0xc0002732c0)
        /go/src/casibase/object/adapter.go:160 +0x3a5
github.com/casibase/casibase/object.CreateTables()
        /go/src/casibase/object/adapter.go:78 +0x3a
main.main()
        /go/src/casibase/main.go:32 +0x3e
panic: SQL logic error: out of memory (1)

goroutine 1 [running]:
github.com/casibase/casibase/object.(*Adapter).createTable(0xc0002ed540)
        /go/src/casibase/object/adapter.go:160 +0x3a5
github.com/casibase/casibase/object.CreateTables()
        /go/src/casibase/object/adapter.go:78 +0x3a
main.main()
        /go/src/casibase/main.go:32 +0x3e
panic: SQL logic error: out of memory (1)

goroutine 1 [running]:
github.com/casibase/casibase/object.(*Adapter).createTable(0xc00028a5c0)
        /go/src/casibase/object/adapter.go:160 +0x3a5
github.com/casibase/casibase/object.CreateTables()
        /go/src/casibase/object/adapter.go:78 +0x3a
main.main()
        /go/src/casibase/main.go:32 +0x3e
```

--- a/object/adapter.go
+++ b/object/adapter.go
@@ -141,7 +141,7 @@ func (a *Adapter) CreateDatabase() error {
 }
 
 func (a *Adapter) open() {
-	engine, err := xorm.NewEngine(a.driverName, a.dataSourceName+a.DbName)
+	engine, err := xorm.NewEngine(a.driverName, conf.GetConfigRealDataSourceName(a.driverName))
 	if err != nil {
 		panic(err)
 	}

